### ISSUE TITLE
Multiple calls of wiki command lead to FetchError

### ISSUE THREAD

**Issue by julianYaman** [julianYaman/wikipedia-bot#37]
**Describe the problem you experienced in as much detail as possible:**
The bug occurs when the `wiki` command get called multiple times at the same time. 
This leads to an `FetchError` and will lead to the Discord reply, that there was a problem executing the command.

**Include here, how to reproduce the problem / bug:**
Calling the wiki command multiple times at the same time.

```js
FetchError: invalid json response body at https://en.wikipedia.org/w/api.php?format=json&action=query&redirects=&prop=extracts&explaintext=&exintro=&titles=Rocket%20League&origin=* reason: Unexpected token < in JSON at position 0
    at D:\Dev\Web-JS\wikipedia-bot\node_modules\node-fetch\lib\index.js:272:32
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  message: 'invalid json response body at https://en.wikipedia.org/w/api.php?format=json&action=query&redirects=&prop=extracts&explaintext=&exintro=&titles=Rocket%20League&origin=* reason: Unexpected token < in JSON at position 0',
  type: 'invalid-json'
}


```

**Possible problem solution:** 
Inserting a cooldown where the user can execute a command 3-5 seconds after the previous one.
This could also prevent spamming and timeouts because of too many API calls more effeciently.

**Details:**
 - OS: All OS affected
 - Node.js version: 12.14.0
 - Version: 1.6.1 - developing 1.7

- [x] I installed the used the bot according to the [contributing guidelines.](https://github.com/julianYaman/wikipedia-bot/blob/master/docs/CONTRIBUTING.md)
- [x] I updated every dependecy to its latest release with `npm update --save`
Branch: `dev-1.7` 
Last commit hash: aaccb3b60d8dc1353414a0ee28396e38a47520aa

**Comment by julianYaman** [comment]
It looks like that this error is common when trying to fetch Wikipedia articles as described on this issue https://github.com/dijs/wiki/issues/136

**Comment by julianYaman** [comment]
The issue will be marked as closed when the 1.7 update will be released!
